Dockerfile

```diff
@@ -18,9 +18,8 @@ COPY --from=builder /app .
 RUN pip install --upgrade pip && \
     pip install -r requirements.txt
 
-RUN python manage.py migrate
-
 EXPOSE 8080
 
-# Run database migrations and start the Django application
-ENTRYPOINT ["python", "manage.py", "runserver", "0.0.0.0:8080"]
+COPY entrypoint.sh /entrypoint.sh
+
+RUN chmod +x /entrypoint.sh
```

Comment:
Add EOF. In order to add EOF add one empty line at the end of the file
![image](https://github.com/mate-academy/devops_todolist_docker_core_task_3_docker_compose/assets/87720583/7ff0d2b2-50dc-49b0-a56b-98cf1251b4f3)